### ISSUE TITLE
Add staticdata which is contained in the python scripts (compile python scripts)

### ISSUE THREAD

**Issue by xforce** [xforce/eve-echoes-tools#2]
Have all the data, just want to write some script to automatically extract it from the python scripts and dump it out into json.
Has some nice stuff

**Comment by FZFalzar** [comment]
Hi, thanks for the tool, I'm trying to rewrite a C# version for it when I'm free, I'm currently developing a quick search API using the data at the moment.

By any chance, do you have any idea where the industry data are located? I can see blueprints, but there is no reference to e.g base number of minerals/items for manufacture.

**Comment by xforce** [comment]
@FZFalzar That is part of that data mentioned in this issue.
Hopefully today or maybe tomorrow I will be able to finally push it.

It is all raw so requires some lookups across a few things for people using it, was thinking about doing some kind of processor to maybe make it a bit easier to work with, but haven't really decided yet if I will do it or not.

**Comment by xforce** [comment]
Specifically the data looks like this 
```        
"10100000101":{
    "blueprint":60100000101,
    "material":{
        "41000000000":38634,
        "41000000002":9642,
        "41000000003":3519,
        "42001000000":701,
        "42001000002":706,
        "42001000005":706,
        "42001000020":211,
        "42001000022":800
    },
    "material_amend_att":1201,
    "minerals_num":[
        38634,
        9642,
        3519
    ],
    "minerals_species":[
        41000000000,
        41000000002,
        41000000003
    ],
    "money":200000,
    "name":"狮鹫级",
    "output_num":1,
    "planetary_material_num":[
        701,
        706,
        706,
        211,
        800
    ],
    "planetary_material_species":[
        42001000000,
        42001000002,
        42001000005,
        42001000020,
        42001000022
    ],
    "product_type_id":10100000101,
    "skill_level":4,
    "time":1000,
    "time_amend_att":1251
},
```

**Comment by FZFalzar** [comment]
Thanks for the insight @xforce ! It would be interesting to know why they split their database like this, instead of putting it all in the  same folders for easier maintenance. I will try to dig into the files as well to locate this and any other info

Not sure if you already know, but the patch ZIP files are accessed using the following scheme:
first 8 hex characters of mmh3_x64(path, 0x9747B28C) + first 8 hex characters of mmh3_x64(path, 0xC82B7479)

The new 16 character hex is then converted to unsigned int64 (ulong), this number is the path to the file

**Comment by FZFalzar** [comment]
Speaking of which, do you happen to know the file name for what you posted? Is it inside an NPK? Or it comes with the OBB? Or one of the EVEM files?

**Comment by xforce** [comment]
Oh yes, I know how they generate the hashes, just wasn't important for anything I do, yet.

As for this issue, scripts are done and pushed.
Added a Dockerimage which can dump all the static data from the Eve Echoes xapk with a single command.
And last but not least, I have updated the S3 bucket with said data, so all there now easy to download. ([v1.5.5 Data](http://eve-echoes-data.s3-website.eu-central-1.amazonaws.com/v1.5.5/raw/))

And the industry data with the blueprint/construction materials can be found [here](https://s3-eu-central-1.amazonaws.com/eve-echoes-data/v1.5.5/raw/py_data/data_common/static/spacestation/industry.json)
